content/en/docs/reference/glossary/manifest.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Manifest
+id: manifest
+date: 2019-06-28
+short_description: >
+  Definition of a Kubernetes API object, serialized into a configuration file (eg JSON or YAML)
+
+aka:
+tags:
+- fundamental
+---
+Manifest is a Kubernetes "API object description" written as config file (JSON or YAML).
+
+<!--more-->
+A manifest defines the desired state of object that Kubernetes will maintain when you apply the manifest. Each configuration file can contain multiple manifests.
```

Comment:
s/configuration file/file/

s/multiple manifests/multiple API objects/

Reply:
I think it's a manifest even if it isn't saved as a file.
Suggestion:
Definition of a Kubernetes API object in JSON or YAML format.

Reply:
Yup, that's better Steve. ++

Reply:
How about:
```suggestion
 You can use a manifest to define an object that Kubernetes will maintain when you apply the manifest. You can define multiple API objects within a single file.
```

I would avoid mentioning _desired state_ here because that term can also refer more narrowly to the `spec` field within an object.

Reply:
You can use a manifest to define an object that doesn't have a desired state, eg Secret. As soon as you define a Secret there's no additional action that needs to happen to make the Secret valid / usable.